eventdisplay/tracks/FairEveTracks.h

```diff
@@ -42,6 +42,7 @@ class FairEveTracks
     Bool_t fUsePt;
     Bool_t fUseEta;
     Bool_t fUseEnergy;
+    Bool_t fAcceptCompound;
```

Comment:
```suggestion
    Bool_t const fAcceptCompound;
```

Let's make it `const` since we only set it in the ctor.

Reply:
You decided against making it `const`?

Reply:
See also https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#S-const. Basically, C++ has the wrong default on constness. I have not checked the rest of your PR for constness opportunities, but I strongly recommend to double check, if you have everything `const` where possible and meaningful.